### PR TITLE
Remove markdown syntax from within React components on History page (en)

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

Replaces the markdown syntax within React components on the History page with HTML. 

Limited to testing this one page (here and on Crowdin) before doing others.

### CODE REVIEW

**Comment by minimalsm** [comment]
@nhsz yes, I believe it makes no difference what we use for as long as it's surrounded by angle brackets.

Would we prefer that?

File: src/content/history/index.md [hunk 1]
```diff
@@ -36,11 +37,13 @@ The Shanghai upgrade brought staking withdrawals to the execution layer. In tand
 
 <ExpandableCard title="Shanghai EIPs" contentPreview="Official improvements included in this upgrade.">
 
-- [EIP-3651](https://eips.ethereum.org/EIPS/eip-3651) – _Starts the `COINBASE` address warm_
-- [EIP-3855](https://eips.ethereum.org/EIPS/eip-3855) – _New `PUSH0` instruction_
-- [EIP-3860](https://eips.ethereum.org/EIPS/eip-3860) – _Limit and meter initcode_
-- [EIP-4895](https://eips.ethereum.org/EIPS/eip-4895) – _Beacon chain push withdrawals as operations_
-- [EIP-6049](https://eips.ethereum.org/EIPS/eip-6049) - _Deprecate `SELFDESTRUCT`_
+<ul>
+<li><a href="https://eips.ethereum.org/EIPS/eip-3651">EIP-3651</a> – <em>Starts the <code>COINBASE</code> address warm</em></li>
```
Comment (minimalsm) [inline src/content/history/index.md:41]: Mmm our linter doesn't run on markdown html

**Comment by nhsz** [comment]
> @nhsz yes, I believe it makes no difference what we use for as long as it's surrounded by angle brackets.
> 
> Would we prefer that?

IMO we should pick one for consistency but also Chakra components have built in a11y that helps so I'd lean to that if it works

**Comment by nhsz** [comment]
> @nhsz yes, I believe it makes no difference what we use for as long as it's surrounded by angle brackets.
> 
> Would we prefer that?

IMO we should pick one for consistency and avoid mixing tags, but also Chakra components have built in a11y that helps so I'd lean to that if it works

**Comment by nhsz** [comment]
> > @nhsz yes, I believe it makes no difference what we use for as long as it's surrounded by angle brackets.
> > Would we prefer that?
> 
> IMO we should pick one for consistency but also Chakra components have built in a11y that helps so I'd lean to that if it works

Didn't know we're already converting html tags into Chakra components, so ok to go with html 👍

**Comment by minimalsm** [comment]
Closing in favour of #10357

FYI, experimenting with lining in this PR but it breaks readability on Crowdin

**Pull request closed without merging**